eventkit_cloud/tasks/export_tasks.py

```diff
@@ -1222,22 +1213,14 @@ def zip_file_task(include_files, run_uid=None, file_name=None, adhoc=False, stat
     # be handled as an ExportTaskRecord.
 
     if not adhoc:
-        # from ..jobs.models import Downloadable
-        run_uid = str(run_uid)
-        if getattr(settings, "USE_S3", False):
-            zipfile_url = s3.upload_to_s3(run_uid, zip_st_filepath, zip_filename)
-        else:
-            if zip_st_filepath != zip_dl_filepath:
-                shutil.copy(zip_st_filepath, zip_dl_filepath)
-            download_url_root = settings.EXPORT_MEDIA_ROOT
-            zipfile_url = os.path.join(download_url_root, run_uid, zip_filename)
-
+        file_size = os.path.getsize(zip_st_filepath) / 1024.0 / 1024.0
```

Comment:
I liked the comment that was removed in the previous line, that said "# MB" to at least give a quick idea what this was trying to obtain.